pkg/cloud/utils.go

```diff
@@ -24,12 +24,18 @@ import (
 	"github.com/GoogleCloudPlatform/k8s-cloud-provider/pkg/cloud/meta"
 )
 
-const (
+var (
 	gaPrefix    = "https://www.googleapis.com/compute/v1"
 	alphaPrefix = "https://www.googleapis.com/compute/alpha"
 	betaPrefix  = "https://www.googleapis.com/compute/beta"
 )
 
+func SetAPIDomain(domain string) {
```

Comment:
Doc string is missing

Reply:
// SetAPIDomain sets the root of the URL for the API. The default domain is
// "https://www.googleapis.com".

Reply:
Whoops, just fixed.